packages/wasm-dpp/src/state_repository.rs

```diff
@@ -723,3 +726,12 @@ impl StateRepositoryLike for ExternalStateRepositoryLikeWrapper {
         bail!("fetching latest platform block failed: value is not number");
     }
 }
+
+fn ctx_to_js_value(maybe_ctx: Option<&StateTransitionExecutionContext>) -> JsValue {
+    if let Some(ctx) = maybe_ctx {
+        let ctx_wasm: StateTransitionExecutionContextWasm = ctx.into();
+        ctx_wasm.into()
+    } else {
+        JsValue::null()
```

Comment:
I think it's better to use `JsValue::undefined` here, as it's more conventional for "none" values in js

Reply:
thanks, I was wondering what I should put here.